dbt/models/qc/docs.md

```diff
@@ -230,5 +230,44 @@ Pulled from the following Inquire queries:
 - `FP Checklist - Card Code Info` (#2160) - Will C.
 {% enddocs %}
 
+# vw_report_town_close_prior_year_card_code_5s_comdat
```

Comment:
**suggestion:** Here too it would be useful to describe the purpose of these tests. What issues are we actually looking for here?

Reply:
Good question! I'm not 100% confident for this particular report, so I've asked Will and I'll adjust this once I get some clarity.

Reply:
I got some clarity from Will on this report and fleshed out the docs in bb8565c3c65622c90657e542b0596a6422aeb7fa.

Reply:
I got some clarity from Will on this report and fleshed out the docs in bb8565c3c65622c90657e542b0596a6422aeb7fa.